projects/plugins/jetpack/tests/php/sync/test_class.jetpack-sync-posts.php

```diff
@@ -854,6 +879,8 @@ public function test_remove_contact_form_shortcode_from_filtered_content() {
 	}
 
 	public function test_remove_likes_from_filtered_content() {
+		$this->markTestSkipped( 'Temp skipping test' );
```

Comment:
I had to mark a few tests as skipped for now, we need to fix those

Reply:
Fixed in https://github.com/Automattic/jetpack/pull/33384/commits/c12fe34362bfdcfe47712bbe1a8d00c2619f3ed5